### ISSUE TITLE
Mouse regression in Life and Death (CGA)

### ISSUE THREAD

**Issue by johnnovak** [dosbox-staging/dosbox-staging#2647]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.81.0-alpha (current main)

### What Operating System are you using?

Mac OS

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Start Life and Death using the provided repro pack. Note you cannot move the mouse past the left half of the screen. This is a regression—the mouse works fine with 0.80.0 in this game.

Retry with CuteMouse (included, see autoexec section). Now the mouse works correctly, but interestingly the pointer also looks different (see screenshot).

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
Repro pack (let me know when I can delete it):

[Life and Death.zip](https://github.com/dosbox-staging/dosbox-staging/files/11868582/Life.and.Death.zip)

**Comment by FeralChild64** [comment]
Cursor range problem should be fixed by PR https://github.com/dosbox-staging/dosbox-staging/pull/2648

**Comment by johnnovak** [comment]
> Cursor range problem should be fixed by PR #2648
> 
> [edit] Cursor shape is a separate problem, though.

Out of interest, are you going to do it? Minor thing and probably very few games are affected, but CuteMouse gets it right.

**Comment by FeralChild64** [comment]
I am going to try - but definitely not for 0.81, this is too risky, it seems to me there is a high risk of regression.

**Comment by johnnovak** [comment]
> I am going to try - but definitely not for 0.81, this is too risky, it seems to me there is a high risk of regression.

Yeah, not worth rushing it for the sake or a slightly crappy CGA game 😄

**Comment by johnnovak** [comment]
> Closing, as the regression got fixed. Created a separate issue for the horizontally doubled mouse cursor.

Thanks @FeralChild64 (for both 😄)
